content/sensu-go/5.21/reference/assets.md

```diff
@@ -91,16 +91,74 @@ For further examples of Sensu users who have added the ability to use a communit
 
 ### Asset path
 
-When you download and install an asset, the asset's local path on disk is exposed to asset consumers via the asset's environment variables.
-This allows you to retrieve the asset's path from a check, hook, handler, or mutator either as an environment variable or using a custom function for token substitution, `assetPath`.
+When you download and install an asset, the asset's local path on disk is exposed to asset consumers via environment variables.
+This allows you to retrieve the asset's path from a check, hook, handler, or mutator either as an environment variable or from a check or hook using a custom function for token substitution, `assetPath`.
 
-For example, if you included a configuration file in the `include` directory of your asset `sensu-go-hello-world-0.0.2`, you can reference the asset from your handler in either of these ways:
+For example, if you included a configuration file in the `include` directory of the asset [`sensu-plugins-windows`][4], you can reference the asset from your handler in either of these ways:
```

Comment:
We should probably update this example to mention a check instead of a handler, since `assetPath` won't be available for handlers and we do not support sensu-backend on Windows.